CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 All changes to `doctl` will be documented in this file.
 
+## [1.20.0] -
```

Comment:
It's traditional to put `UNRELEASED` instead of the date. Not going to hold the PR for it, though :)